src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,50 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("Starting the process of completing an order for userId: {}", userId);
+        List<Product> products = null;
+        logger.info("Retrieving products from the shopping cart");
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
+                    userId, products);
+        } catch (Exception e) {
+            logger.error("Failed to retrieve products from shopping cart for userId: {}",
+                    userId, e);
+            throw new RuntimeException("Please try again");
+        }
+
         Order order = new Order(products, userId);
+        logger.debug("Order created for userId: {} with products: {}", userId, products);
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
```

Comment:
Hardcoding the order ID as 1L is not a good practice. The order ID should be generated in a way that ensures it is unique for each order.